splice/static/css/spliceApp.css

```diff
@@ -97,10 +97,48 @@ ol.tiles-list > li:before {
 }
 
 .tiles-info {
-    border-top: 1px solid #DDD;
     padding-top: 20px;
+    background-color: #FFF;
+}
+
+.channel-controls {
+    margin-bottom: 25px;
+}
+
+.distribution-controls {
+    margin-bottom: 25px;
+}
+
+.distribution-previous-select {
+    clear: both;
+    display:block;
+}
+
+.distribution-file-input {
+    border: 1px solid #DDD;
+    padding: 5px;
+    border-radius: 2px;
+}
+
+.modal-backdrop {
+    opacity: 0.7;
+}
+
+.modal-status-message {
+    word-wrap: break-word;
+    overflow-y: auto;
+    max-height: 700px;
```

Comment:
This is as designed. I do not think anyone will want to view those links in that UI. There's a feature to easily copy/paste the content in the status message in case one wants to look at the data in depth